Department.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Main;
+
+use ArrayObject;
+
+class Department
+{
+    private string $name;
+    private array $workers;
```

Comment:
```suggestion
    private ArrayObject $workers; // added type hinting for workers property
    
```
- added type hinting for workers property